pyproject.toml

```diff
@@ -61,15 +62,49 @@ documentation = "https://docs.easyreflectometry.org"
 [tool.hatch.metadata]
 allow-direct-references = true
 
+[tool.hatch.build.targets.wheel]
+packages = ["EasyReflectometry"]
+
 [tool.coverage.run]
 source = ['EasyReflectometry']
 
 [tool.github.info]
 organization = 'easyScience'
 repo = 'EasyReflectometryLib'
 
-[tool.hatch.build.targets.wheel]
-packages = ["EasyReflectometry"]
+[tool.ruff]
+line-length = 88
```

Comment:
Just like with the `isort` PR - does it make sense to limit the line width to 88? Removed `flake8` formatted uses 127.

Reply:
changed to 127